Sources/ProjectSpec/Project.swift

```diff
@@ -173,7 +173,17 @@ extension Project {
         let jsonDictionary = Project.resolveProject(jsonDictionary: jsonDictionary)
 
         name = try jsonDictionary.json(atKeyPath: "name")
-        settings = jsonDictionary.json(atKeyPath: "settings") ?? .empty
+
+        do {
+            settings = try jsonDictionary.json(atKeyPath: "settings")
+        } catch let specParsingError as SpecParsingError {
```

Comment:
```suggestion
        } catch specParsingError is SpecParsingError {
```

`is` is enough if there is no typed throw

Reply:
When using `catch is SpecParsingError ` form, the error instance is not available within the block.
So, I’ll keep using let specParsingError as SpecParsingError.